truffle-config.js

```diff
@@ -1,5 +1,38 @@
+const path = require('path');
+
+const { NearProvider } = require('near-web3-provider');
+
+function NearLocalProvider() {
+    return new NearProvider({
+        nodeUrl: 'http://127.0.0.1:3030',
+        networkId: 'balancer-tests',
+        masterAccountId: 'test.near',
+        keyPath: path.join(process.env.HOME, '.near/local/validator_key.json'),
+    });
+}
+
+function NearTestNetProvider() {
+    return new NearProvider({
+        nodeUrl: 'http://34.82.212.1:3030',
+        networkId: 'default',
+        masterAccountId: '1.test.near',
+        evmAccountId: 'evm',
+        keyPath: path.join(__dirname, '1.test.near.json'),
+    });
```

Comment:
This credentials and nodeUrl should be replaced with betanet?

Reply:
In theory we should add betanet instructions here. But running tests on betanet is harder as requires to actually setup accounts there. 
Having CLI to setup a bunch of test accounts seems like a really good idea then - so you can do it on whatever network you are testing with.

Reply:
updated readme to reflect just using betanet